common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -8707,6 +8707,16 @@
       "Doesn't support month or year interval: <interval>"
     ]
   },
+  "_LEGACY_ERROR_TEMP_3263" : {
```

Comment:
Will we reassign a more reasonable name for `_LEGACY_ERROR_TEMP_[3263|3264]` in the future? Or will this name be kept for a long time?

Reply:
Yep, need to assign proper names + tests + avoid general embedded `message` (which will require dozens new error conditions). I leave it as is because the last one need a lot of work.